Graphs/AbstractGraph.js

```diff
@@ -260,8 +264,9 @@ export default class AbstractGraph extends React.Component {
             return;
 
         const scale = d3.scaleOrdinal([...colors, ...otherColors]);
-        scale.domain(data.map((d) => d[colorColumn || defaultColumn]));
+        const domainData = d3.map(data, (d) => d[colorColumn || defaultColumn]).keys().sort();
 
+        scale.domain((setGraphColor && setGraphColor(domainData)) || domainData);
```

Comment:
This is great it gives a the same color scheme however what I think we are looking for is a mapping between runtime values and colors, as per your example: `{ "WEBEX": "#D9070A", "NETFLIX": "#FFBF02", "CISCOVPN": "#62AC00" }`